# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mine/new_dialog_version.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./selector.c.orig	2005-02-09 13:13:48.000000000 -0300
+++ ./selector.c	2005-02-09 13:24:01.514859312 -0300
@@ -18,6 +18,7 @@
  */
 
 #include "gasgui.h"
+#define __DIALOG_OLD_NAMES__
 #include <dialog.h>
 #include <limits.h>
 #include <sys/ioctl.h>
@@ -535,7 +536,7 @@
 		screensize_y = ws.ws_row - 3;
 	}
 
-	init_dialog();
+	init_dialog(stdin,stdout);
 
 	/* Yup - that's a label for goto.... */
 main_loop:
